lib/promotion.rb

```diff
@@ -11,6 +11,9 @@ def initialize(user)
   # Review a user for a promotion. Delegates work to a review_#{trust_level} method.
   # Returns true if the user was promoted, false otherwise.
   def review
+    result = DiscoursePluginRegistry.apply_modifier(:review_trust_level, @user)
```

Comment:
I wonder if it'd be bettter to have 
```ruby
override = DiscoursePluginRegistry.apply_modifier(:review_trust_level, @user)
return if override
```
Where we return `true` from a `register_modifier` block, if the review process is overriden, and return `nil` or `false` if the normal flow should continue